templates/news-detail/news-detail.js

```diff
@@ -0,0 +1,78 @@
+/* eslint-disable function-call-argument-newline, object-curly-newline, function-paren-newline */
+import { aside, div, a, button, strong, small } from '../../scripts/dom-helpers.js';
+import { createOptimizedPicture, getMetadata } from '../../scripts/aem.js';
+import { loadFragment } from '../../blocks/fragment/fragment.js';
+
+export default async function decorate(doc) {
+  const $page = doc.querySelector('main .section');
+  const $mainContent = div();
+  const articleChildren = [...$page.children];
+  [...articleChildren].forEach((child) => {
```

Comment:
This destructuring doesn't seem necessary. Could you do forEach directly on `$page.children`?

Reply:
Good catch - I pulled this from another site - but it can definitely be simplified.